pkg/compare/compare.go

```diff
@@ -455,26 +472,49 @@ func (o *Options) Run() error {
 			clusterObj:              &clusterCR,
 			FieldsToOmit:            o.ref.FieldsToOmit,
 		}
+		crName := apiKindNamespaceName(&clusterCR)
+		if patches, ok := o.patches[crName]; ok {
```

Comment:
why are the patches mapped by a specific CR, instead of the corresponding reference.
it can be more stable to changes in the CRS name-namespace-apiversion-kind. and its logical that you would like to supresss a diff because of a field in the ref.